app/ante_no_seq.go

```diff
@@ -66,5 +78,11 @@ func NewAnteHandler(
 		ante.NewSigVerificationDecorator(ak, signModeHandler),
 		// ante.NewIncrementSequenceDecorator(ak),
 		ibcante.NewRedundantRelayDecorator(channelKeeper),
+		// auction module antehandler
```

Comment:
Why do we not have classicSignatureVerificationDecorator and authenticatorVerificationDecorator in this file as well?